flask_saml_sso/session.py

```diff
@@ -1,8 +1,31 @@
+import enum
 from datetime import datetime
+from datetime import timedelta
 
+import flask
 from flask_session import sessions
 from itsdangerous import BadSignature, want_bytes
 
+LOGGED_IN = 'loggedIn'
+SAML_SESSION_INDEX = 'samlSessionIndex'
+SAML_NAME_ID = 'samlNameId'
+SAML_ATTRIBUTES = 'samlAttributes'
+SAML_SESSION_TYPE = 'samlSessionType'
+
+
+class SessionType(enum.Enum):
+    User = 1
+    Service = 2
```

Comment:
If this ends up in the database, wouldn't string keys be cleaner?

Reply:
It ends up pickled in the database, so it shouldn't make a difference.